Engine/source/gui/3d/guiTSControl.cpp

```diff
@@ -578,14 +578,12 @@ void GuiTSCtrl::onRender(Point2I offset, const RectI &updateRect)
       GFXTextureObject *texObject = mStereoGuiTarget->getTexture(0);
       const FovPort *currentFovPort = GFX->getStereoFovPort();
       const MatrixF *eyeTransforms = GFX->getStereoEyeTransforms();
-      const MatrixF *worldEyeTransforms = GFX->getInverseStereoEyeTransforms();
       const Point3F *eyeOffset = GFX->getStereoEyeOffsets();
 
       for (U32 i=0; i<2; i++)
       {
          GFX->activateStereoTarget(i);
          Frustum gfxFrustum = originalFrustum;
-         const F32 frustumDepth = gfxFrustum.getNearDist();
```

Comment:
What would you preffer be done with https://github.com/GarageGames/Torque3D/blob/6b9e5dd15e67b152500a3ed0e3e58c695d2c959d/Engine/source/gui/3d/guiTSControl.cpp#L621 in that case?